packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -320,12 +320,12 @@ class Doctor {
   Future<bool> diagnose({
     bool androidLicenses = false,
     bool verbose = true,
-    bool showColor = true,
```

Comment:
to be consistent with the rest of the tool, we should check if the terminal supports color, rather than creating a special function parameter only for use in tests